osu.Game.Rulesets.Osu/UI/Cursor/OsuCursorContainer.cs

```diff
@@ -31,7 +31,7 @@ public partial class OsuCursorContainer : GameplayCursorContainer, IKeyBindingHa
 
         private readonly Bindable<bool> showTrail = new Bindable<bool>(true);
 
-        private readonly Drawable cursorTrail;
+        public readonly Drawable CursorTrail;
```

Comment:
i dont think this will fly, theres no way mods should be accesing the cursor directly, check other mods, i dont think theres a single one that access whatever its changing in this fashion